code/modules/mob/living/emote.dm

```diff
@@ -326,7 +326,7 @@
 /datum/emote/living/surrender
 	key = "surrender"
 	key_third_person = "surrenders"
-	message = "puts their hands on their head and falls to the ground, they surrender%s!"
+	message = "puts their hands on their head and falls to the ground, they surrender!"

```

Comment:
Would it be extra specially bettererers if you do the gender his/her thing instead of their?

I assume that exists...

And also do the same for they.


I mean, what if you're in a situation where a man, a woman and a brought to life drinks machine are in a mexican standoff? If one shoots, they all shoot but they can't kill both of their opponents. So one might try to surrender, not because they believe they won't get shot but because the other two might kill each other if they think the third isn't a threat.

All is fine and dandy because you can just read the name at the start of the message and the word surrender at the end.

But wait, what's this? They each examine each other. They all look unnerved. *Everyone is named Dr. Gibb drinks machine, **now how will they know who surrenders?!***


Save mexican stand offs, genderify the emote.